server/src/main/java/dev/slimevr/vr/trackers/VRTracker.java

```diff
@@ -23,6 +27,54 @@ public VRTracker(int id, String name, boolean hasRotation, boolean hasPosition)
 		super(id, name, name, hasRotation, hasPosition, null);
 	}
 
+	@Override
```

Comment:
I'd love to at least have some kinda note about this code duplication for future reference
```suggestion
	// TODO: Reduce code duplication from IMUTracker.java
	@Override
```

Reply:
Well, I can add it, but we plan to refactor all of the classes into a single one, like discussed with Futura

Reply:
Added both notes in the end